perun-db/postgres.sql

```diff
@@ -1,4 +1,4 @@
--- database version 3.1.65 (don't forget to update insert statement at the end of file)
+-- database version 3.1.66 (don't forget to update insert statement at the end of file)
```

Comment:
You should also remove "suspendedTo" from members table.

Reply:
Done.